scripts/migrate_to_rapidpro/sync_fallback_channel.py

```diff
@@ -0,0 +1,123 @@
+import json
+import time
+from urllib.parse import urljoin
+
+import psycopg2
+import requests
+
+DB = {
+    "dbname": "ndoh_rapidpro_contacts",
+    "user": "postgres",
+    "port": 5432,
+    "host": "localhost",
+}
+
+TURN_URL = "http://localhost:4000"
+TURN_TOKEN = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJFbmdhZ2VkIiwiZXhwIjoxNjIyMTI0MjUxLCJpYXQiOjE1OTA1ODgyNjAsImlzcyI6IkVuZ2FnZWQiLCJqdGkiOiI4ZWNmMTYzNi1iMjI2LTQxMmEtYjNlYi1jZjI2ZGQ0M2VhYWQiLCJuYmYiOjE1OTA1ODgyNTksInN1YiI6Im51bWJlcjoyIiwidHlwIjoiYWNjZXNzIn0.9CuZHN84SIDiiSyChXWKDXeRU3Nl27s5HysWP38ck1QUihRXuaadViDVlIOXr6QRosTzEHqEo4hsGntsajjAjA"  # noqa
+
+
+def get_turn_contact_details(wa_id):
+    headers = {
+        "Authorization": f"Bearer {TURN_TOKEN}",
+        "content-type": "application/json",
+        "Accept": "application/vnd.v1+json",
+    }
+    response = requests.get(urljoin(TURN_URL, f"/v1/contacts/{wa_id}"), headers=headers)
```

Comment:
Should we do a `response.raise_for_status()`, to make sure that the status code is correct?